src/main/scala/SparkEvents.scala

```diff
@@ -70,15 +70,11 @@ object SparkEvents {
     }
   }
 
-  def writeEventsToFile(
-      inputStream: InputStream,
-      outputPath: Path,
-      hadoopConf: Configuration): Unit = {
+  def writeEventsToFile(inputStream: InputStream, outputPath: Path, hadoopConf: Configuration): Unit = {
     val fs = outputPath.getFileSystem(hadoopConf)
     val fos = fs.create(outputPath)
     try {
       val buf = new Array[Byte](8192)
-      var len = inputStream.read(buf)
```

Comment:
This is the single line change I wanted to make - ended up being a massive PR :-)

Reply:
Just seeing this PR :) Sorry for the delayeddddd response :-)